src/main/java/in/projecteka/datanotificationsubscription/clients/model/Session.java

```diff
@@ -0,0 +1,31 @@
+package in.projecteka.datanotificationsubscription.clients.model;
+
+import com.fasterxml.jackson.annotation.JsonAlias;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Builder
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class Session {
```

Comment:
There will already be a Session Model. If possible we can use that itself.

Reply:
Done